docs/release-process.md

```diff
@@ -0,0 +1,11 @@
+Martha Release Process
+=========
+
+Martha's release process is documented and tracked through a Jira checklist. 
+
+Martha is updated rarely enough that we typically include a release in the 
+work of each ticket. Navigate to the ticket associated with the work you're 
+releasing, or create a new ticket for the release. In the Jira ticket, click 
+on Add Checklist, then click on the button with three dots in the upper right 
```

Comment:
"Add Checklist" is this button, right?
![Screen Shot 2022-04-14 at 3 07 52 PM](https://user-images.githubusercontent.com/19228696/163459630-a3adb087-3d0b-42bc-b9a6-4bb61787df68.png)

Reply:
Yes! In other views of the ticket there's text in it. I'll expand the description here.